File: Common-Modules/src/main/java/com/common/dto/SalonResponseDTO.java
package com.common.dto;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class SalonResponseDTO extends BaseResponseDTO {
	private String saloonName;
	private String address;
	private String phone;
	private Boolean active;
	private List<SaloonServiceDTO> services;
	private String createdBy;
}

File: src/app/services/DTOs/salon-response-dto.ts
import { SaloonServiceDTO } from "./saloon-service-dto";

export interface SalonResponseDTO {
    id: string;
    saloonName: string;
    address: string;
    phone: string;
    active: boolean;
    services: SaloonServiceDTO[];
    createdBy: string;
    createdAt: string;
    updatedAt: string;
}
